src/components/form/form/Questions.jsx

```diff
@@ -56,6 +56,13 @@ const Questions = ({
     onSubmit(setLoading, setState);
   };
 
+  const handleClick = (user, option, field, setUser) => {
```

Comment:
user, and setUser shouldn't have to be parameters if they alr exist within the parent component